release-notes/7.0/supported-os.md

```diff
@@ -28,13 +28,13 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
-[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS][CentOS]                      | 7                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Stream][CentOS]               | 8+                    | x64               | [CentOS][CentOS-lifecycle]
 [Debian][Debian]                      | 10+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
 [Fedora][Fedora]                      | 36+                   | x64               | [Fedora][Fedora-lifecycle]
 [openSUSE][OpenSUSE]                  | 15+                   | x64               | [OpenSUSE][OpenSUSE-lifecycle]
-[Oracle Linux][Oracle-Linux]          | 7+                    | x64               | [Oracle][Oracle-lifecycle]
-[Red Hat Enterprise Linux][RHEL]      | 7+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
+[Oracle Linux][Oracle-Linux]          | 8+                    | x64               | [Oracle][Oracle-lifecycle]
+[Red Hat Enterprise Linux][RHEL]      | 8+                    | x64, Arm64        | [Red Hat][RHEL-lifecycle]
 [SUSE Enterprise Linux (SLES)][SLES]  | 12 SP2+               | x64               | [SUSE][SLES-lifecycle]
```

Comment:
Not fully related but just out of curiosity:
`12 SP2+` in this case really only means `12 SP2+` & `15`.

Here are the support times for SLES:
![image](https://user-images.githubusercontent.com/940619/231376403-8497e9e0-2597-453e-aa1b-3ad4ce6755ee.png)
![image](https://user-images.githubusercontent.com/940619/231376533-b9c2cbf2-f7d8-4982-9f7f-4667cc48138e.png)

So I would expect the following as of now: `12 SP4+, 15 SP1+`

Reply:
We should also try to reflect changes for version that haven't been only included by fault in the `Out of support OS versions` section. So for .NET 6 and .NET 7 this would include SLES 12 SP2, SLES 12 SP3 and SLES 15 (unpatched).